### ISSUE TITLE
Update pes_learning_rate to accept filter

### ISSUE THREAD

**Issue by arvoelke** [arvoelke/nengolib#134]
http://compneuro.uwaterloo.ca/publications/voelker2017c.html

We can update the function (or provide a new one) to take a linear filter and suggest an upper-bound for the learning rate (to avoid oscillations).
